### PR TITLE
Feature/liayoo/admin apis

### PR DESCRIPTION
**Pull request by liayoo**
- Renamed the api-access to admin json-rpc apis
- Fixed the `clean` npm script

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "node": ">=14"
   },
   "scripts": {
-    "clean": "sh stop_servers_local.sh & echo 'cleaned'",
+    "clean": "sh stop_local_blockchain.sh & echo 'cleaned'",
```
Comment (platfowner) [inline package.json:16]: what about s/clean/stop/?

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks!

File: package.json (hunk 1 above)
Comment (liayoo) [reply package.json]: Sounds good! Thanks for the suggestion. It's been updated.

**Comment by liayoo** [comment]
Thanks for the fast review! Now merging~

**Pull request merged**